frontend/src/metabase-lib/expressions/__support__/expressions.js

```diff
@@ -6,104 +6,118 @@ import {
-            createMockMetric({
-              id: 1,
-              name: "metric",
-            }),
-            createMockMetric({
-              id: 2,
-              name: "metric",
-            }),
-          ],
+export const DEFAULT_QUERY = {
+  database: SAMPLE_DATABASE.id,
+  type: "query",
+  query: {
+    "source-table": TABLE_ID,
+  },
+};
+
+const database = createMockDatabase({
+  id: SAMPLE_DB_ID,
+  name: "db",
+  tables: [
+    createMockTable({
+      db: DB_ID,
+      id: TABLE_ID,
+      fields: [
+        createMockField({
+          id: 1,
+          table_id: TABLE_ID,
+          display_name: "A",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 2,
+          table_id: TABLE_ID,
+          display_name: "B",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 3,
+          table_id: TABLE_ID,
+          display_name: "C",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 10,
+          table_id: TABLE_ID,
+          display_name: "Toucan Sam",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 11,
+          table_id: TABLE_ID,
+          display_name: "Sum",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 12,
+          table_id: TABLE_ID,
+          display_name: "count",
+          base_type: TYPE.Float,
+        }),
+        createMockField({
+          id: 13,
+          table_id: TABLE_ID,
+          display_name: "text",
+          base_type: TYPE.Text,
+        }),
+        createMockField({
+          id: 14,
+          table_id: TABLE_ID,
+          display_name: "date",
+          base_type: TYPE.DateTime,
         }),
       ],
-      features: [
-        "basic-aggregations",
-        "standard-deviation-aggregations",
-        "expression-aggregations",
-        "percentile-aggregations",
-        "foreign-keys",
-        "native-parameters",
-        "expressions",
-        "advanced-math-expressions",
-        "right-join",
-        "left-join",
-        "inner-join",
-        "nested-queries",
-        "advanced-math-expressions",
+      segments: [
+        createMockSegment({
+          id: 1,
+          name: "segment",
+          table_id: TABLE_ID,
+        }),
+      ],
+      metrics: [
+        createMockMetric({
+          id: 1,
+          name: "metric",
+          table_id: TABLE_ID,
+        }),
+        createMockMetric({
+          id: 2,
+          name: "metric",
+          table_id: TABLE_ID,
+        }),
       ],
     }),
   ],
+  features: [
+    "basic-aggregations",
+    "standard-deviation-aggregations",
+    "expression-aggregations",
+    "percentile-aggregations",
+    "foreign-keys",
+    "native-parameters",
+    "expressions",
+    "advanced-math-expressions",
+    "right-join",
+    "left-join",
+    "inner-join",
+    "nested-queries",
+    "advanced-math-expressions",
```

Comment:
This is specified twice in the array (I noticed it when I tried to see how difficult it is to convert this file to TS).

Reply:
thanks! please take a look https://github.com/metabase/metabase/pull/36265